modules/serverless-kn-container.adoc

```diff
@@ -0,0 +1,60 @@
+[id="serverless-kn-container_{context}"]
+= Knative client multi-container support
+
+You can use the `kn container add` command to print YAML container spec to standard output. This command is useful for multi-container use cases because it can be used along with other standard `kn` flags to create definitions. The `kn container add` command accepts all container-related flags that are supported for use with the `kn service create` command. The `kn container add` command can also be chained through Unix pipes (`|`) to create multiple container definitions at once.
+
+.Example commands
+
+* Add a container from an image and print it to standard output:
++
+[source,terminal]
+----
+$ kn container add <container_name> --image <image_uri>
+----
++
+.Example command
+[source,terminal]
+----
+$ kn container add sidecar --image docker.io/example/sidecar
+----
++
+.Example output
+[source,terminal]
+----
+containers:
+- image: docker.io/example/sidecar
+  name: sidecar
+  resources: {}
+----
+
+* Chain two `kn container add` commands together, then pass them to a `kn service create` command, to create a Knative service with two containers:
```

Comment:
> together, then pass them

"together, and then pass them"

ISG recommends using "Do this, and then do that" (preferred) or "Do this; then do that." Does it make sense to move the infinitive clause to the beginning of the sentence?

Reply:
If you decide to leave the infinitive clause at the end, I think you can cut the last comma.

Reply:
Updated, thanks!